tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -8405,7 +8465,14 @@ def localWriteSwapOffsets(self, kernel, internalPointerSwap, tP):
                 src1=sgpr("LocalWriteAddr%s"%tPM["tensorChar"]), \
                 comment="swap Red Blk SGPR"))
           else:
-            numLwa = self.states.m.numVgprLocalWriteAddr
+            numLwa = 0;
```

Comment:
I think this part of the code is for metadata only. No need to add A/B code here.